data/compliance/pages.json

```diff
@@ -0,0 +1,97 @@
+[
+    {
+        "framework": "PCI DSS",
+        "cloud": "AWS",
+        "description": "PCI DSS (Payment Card Industry Data Security Standard) compliance refers to the adherence to a set of security standards designed to protect card information during and after a financial transaction. These standards are established by the Payment Card Industry Security Standards Council (PCI SSC), which was founded by major credit card companies like Visa, MasterCard, American Express, Discover, and JCB.",
```

Comment:
I want this definition to be after the CTA. 

I want it to be part of section that says "What is PCI DSS Compliance?"

In place of this paragraph under "PCI DSS Compliance for AWS", I want to add some copy that says something like. 

PCI DSS compliance is critical to protecting cardholder data that is processed, stored, and transmitted. Pulumi can assist you with making your AWS cloud infrastructure PCI DSS. Pulumi can help you identify existing cloud resources that are not in compliance, and it can also enforce compliance policies proactively before infrastructure is deployed. Get started with Pulumi to use these compliance tools or speak with a Solutions Architect to get a expert consultation.

Reply:
The above comment was for the Compliance for Cloud page

We may want a slightly different format for the cloud resource pages